build/transpileWS.js

```diff
@@ -147,81 +147,6 @@ class CCXTProTranspiler extends Transpiler {
         return false
     }
 
-    // ------------------------------------------------------------------------
-
-    transpileOrderBookTest () {
-        const jsFile = './ts/src/pro/test/base/test.OrderBook.ts'
-        const pyFile = './python/ccxt/pro/test/test_order_book.py'
-        const phpFile = './php/pro/test/OrderBook.php'
-        const pyImports = [
-            '',
-            'from ccxt.async_support.base.ws.order_book import OrderBook, IndexedOrderBook, CountedOrderBook  # noqa: F402',
-            '',
-        ].join ('\n')
-        this.transpileTest (jsFile, pyFile, phpFile, pyImports)
-    }
-
-    // ------------------------------------------------------------------------
-
-    transpileCacheTest () {
-        const jsFile = './ts/src/pro/test/base/test.Cache.ts'
-        const pyFile = './python/ccxt/pro/test/test_cache.py'
-        const phpFile = './php/pro/test/Cache.php'
-        const pyImports = [
-            '',
-            'from ccxt.async_support.base.ws.cache import ArrayCache, ArrayCacheByTimestamp, ArrayCacheBySymbolById, ArrayCacheBySymbolBySide  # noqa: F402',
-            '',
-        ].join ('\n')
-        this.transpileTest (jsFile, pyFile, phpFile, pyImports)
-    }
-
-    // ------------------------------------------------------------------------
-
-    transpileTest (jsFile, pyFile, phpFile, pyImports) {
-
-        log.magenta ('Transpiling WS from', jsFile.yellow)
-
-        let js = fs.readFileSync (jsFile).toString ()
-
-        js = this.regexAll (js, [
-            [ /\'use strict\';?\s+/g, '' ],
-            //[ /[^\n]+require[^\n]+\n/g, '' ],
-            [ /[^\n]+from[^\n]+\n/g, '' ],
-            [ /\/\* eslint-disable \*\/\n*/g, '' ],
-            [ /export default\s+[^\n]+;*\n*/g, '' ],
-            [ /function equals \([\S\s]+?return true;?\n}\n/g, '' ],
-        ])
-
-        const options = { js, removeEmptyLines: false }
-        const transpiled = this.transpileJavaScriptToPythonAndPHP (options)
-        const { python3Body, python2Body, phpBody } = transpiled
-        const pythonHeader = [
-            '',
-            '',
-            'def equals(a, b):',
-            '    return a == b',
-            '',
-        ].join ('\n')
-
-        const phpHeader = [
-            '',
-            'function equals($a, $b) {',
-            '    return json_encode($a) === json_encode($b);',
-            '}',
-        ].join ('\n')
-
-        const python = this.getPythonPreamble () + pyImports + pythonHeader + python2Body
-        const php = this.getPHPPreamble () + phpHeader + phpBody
-
-        log.magenta ('→', pyFile.yellow)
-        log.magenta ('→', phpFile.yellow)
-
-        overwriteFile (pyFile, python)
-        overwriteFile (phpFile, php)
-    }
-
-    // ------------------------------------------------------------------------
```

Comment:
why was this removed?

Reply:
all test files are being transpiled from transpile.js from now. we will have many other test files in the future, so listing them one by one and dedicating them separate methods is a 'no-go' imo.

Reply:
The thing is I'm not sure we should mix these tests transpiling with the normal tests, even in the c# side I kept them separated. each test type has different needs and keeping everything within the same method with a lot of (if isWsCacheTest) etc gets confusing very quickly

Reply:
but as we use Ast transpiler, it doesn't have much difference whether the transpiling file is from 'examples' folder, from 'tests', from 'rest' exchagnes or from 'ws' exchanges. if we separate the logic, then we shuold have `transpile-examples, transpile-tests, transpile-rest, transpile-ws' methods, and on top of that, `transpile-cache, transpile-orderbook, transpile-xyz...` which is more hard to maintain and track. when doing this from same `transpileTests` method, it's easier to control and manage everything

Reply:
so, it actually does not matter for transpiler whether the file being transpiled is from ws folder or from elsewhere.
but if you say otherwise, i will try to tear down them and migrate ws parts...

Reply:
we can do everything every from transpile tests but I would keep it in different methods like we have now, like
```
transpileCacheTest()
transpileOrderBookTest()
```
In the future if we need to apply a specific fix it will be much easier

Reply:
@ttodua can you please keep the context-specific tests (orderbook, cache, etc) in their isolated methods, as they were before? As I said I don't think that mixing them is a good idea just makes thing more confusing

Reply:
ok, i'll try to revert.

Reply:
But they're fundamentally different tests and at any time we might need to apply specific fixes to them

Reply:
if everything is in a single place it will be much harder

Reply:
the thing is that the code is being transpiled well and any update would not cause much need to do language-specific complex edits (because we have unified tests & checks almost for everyting, and still increasing details & coverage), so, i was forwarding to avoid separate methods like "transpile_test_file_A, transpile_test_file_B," and etc.. 
but as you say, i'll do so ✌️